RecoMuon/DetLayers/plugins/MuonDetLayerGeometryESProducer.cc

```diff
@@ -90,3 +107,11 @@ std::unique_ptr<MuonDetLayerGeometry> MuonDetLayerGeometryESProducer::produce(co
 
   return muonDetLayerGeometry;
 }
+
+void MuonDetLayerGeometryESProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  //no parameters are used
```

Comment:
Is fillDescriptions still needed/useful, then?

Reply:
Yes. This tells the framework that no parameters are allowed (which is correct for this module). If one does not provide a `fillDescriptions`, that tells the framework any parameters are allowed.